usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -365,21 +362,12 @@ def handle(self, *args, **options):
             with timer("insertion of new FPDS data in batches", logger.info):
                 self.insert_all_new_fpds(total_insert)
 
-            # Update Awards based on changed FPDS records
-            with timer("updating awards to reflect their latest associated transaction info", logger.info):
-                update_awards(tuple(AWARD_UPDATE_ID_LIST))
-
             # Update FPDS-specific Awards based on the info in child transactions
             with timer("updating contract-specific awards to reflect their latest transaction info", logger.info):
                 update_contract_awards(tuple(AWARD_UPDATE_ID_LIST))
```

Comment:
In this new version, update_contract_awards now occurs BEFORE update awards. This didn't cause any errors, but it might have changed something via side effects.

Reply:
I can't find side-effects as first glance, but don't see a reason that prevents `update_contract_awards()`being after `upsert_transactions()`

Reply:
putting it after upsert would also change functionality, since now it's running after the update_transactions and update_c_d_links is called. The other option is to put it in the upsert_transactions() call and use a flag to only have this happen in fpds cases, but that seems un-SOLID and risky in its own way.